packages/manager-components/src/components/content/price/price.component.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_TEXT_COLOR_HUE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import {
+  OvhSubsidiary,
+  ovhSubsidiaryToCurrency,
+  CentsUnit,
+  Frequencytype,
+} from '../../../enumTypes';
+
+export interface PriceProps {
+  price: number;
+  frequency?: Frequencytype;
+  unit?: CentsUnit;
+  htOnly?: boolean;
+  withTTC?: boolean;
+  includeGST?: boolean;
+  excludeGST?: boolean;
+  htWithTTC?: boolean;
+  ovhSubsidiary?: OvhSubsidiary;
```

Comment:
you have 3 props : htOnly, withTTC, htWithTTC
i think it would be simpler to have in props : isHT, isTTC

it's the same logic and you can express you 3 previous props this way : 
 - htOnly => isHT=true && isTTC=false
 - withTTC => isHT=false && isTTC=true
 - htWithTTC => isHT=true && isTTC=true

also your htOnly props is misleading as setting both htOnly and withTTC will display ttc price (so it's not ht only)